src/status_im/ui/screens/hardwallet/pin/views.cljs

```diff
@@ -10,6 +10,8 @@
             [status-im.utils.platform :as platform]
             [status-im.ui.components.topbar :as topbar]))
 
+(def ^:const default-pin-retries-number 3)
```

Comment:
why not in constants ns ?

Reply:
until it is need anywhere else I see no point in making it "global"